### PR TITLE
Initiator - tests

### PR DESCRIPTION
**Pull request by Itay4**
## Status
- [x] Ready

## Description
Added tests for `init`:
 - `get_created_dir_name`
 - `get_object_id`
 - `create_metadata`
 - `get_valid_user_input`
 - `create_new_directory`

### CODE REVIEW

File: demisto_sdk/commands/init/tests/initiator_test.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+        'description': '## FILL OUT MANUALLY ##',
+        'support': 'demisto',
+        'serverMinVersion': '## FILL OUT MANUALLY #',
+        'currentVersion': PACK_INITIAL_VERSION,
+        'author': 'demisto',
+        'url': 'https://www.demisto.com',
+        'email': '',
+        'categories': [],
+        'tags': [],
+        'created': datetime.utcnow().strftime(Initiator.DATE_FORMAT),
```
Comment (yaakovi) [inline demisto_sdk/commands/init/tests/initiator_test.py:78]: could be flaky in edge case
